website/docs/r/iam_role.html.markdown

```diff
@@ -49,6 +49,8 @@ The following arguments are supported:
   See [IAM Identifiers](https://docs.aws.amazon.com/IAM/latest/UserGuide/Using_Identifiers.html) for more information.
 * `description` - (Optional) The description of the role.
 
+* `max_session_duration` - (Optional) Max duration of a federated role 
```

Comment:
This should probably note that the duration is in seconds and defaults to 3600. The [SDK documentation](https://docs.aws.amazon.com/sdk-for-go/api/service/iam/#CreateRoleInput) sums it up pretty well:
> The maximum session duration (in seconds) that you want to set for the specified role. If you do not specify a value for this setting, the default maximum of one hour is applied. This setting can have a value from 1 hour to 12 hours.